README.rst

```diff
@@ -4,24 +4,16 @@ edx-rbac
 |pypi-badge| |travis-badge| |codecov-badge| |doc-badge| |pyversions-badge|
 |license-badge|
 
-The ``README.rst`` file should start with a brief description of the repository,
-which sets it in the context of other repositories under the ``edx``
-organization. It should make clear where this fits in to the overall edX
-codebase.
-
-Library to help managing role based access controls for django apps
-
 Overview
 --------
 
-Library to help manage role based access controls for django apps.
-See the `Getting started <https://github.com/edx/edx-rbac/blob/master/docs/getting_started.rst>`_ guide to begin developing.
-
+Library to help manage role based access controls for django services.
 
-Documentation
--------------
+* See the `Getting started <https://github.com/edx/edx-rbac/blob/master/docs/getting_started.rst>`_ guide to setup
+your development environment.
```

Comment:
Looking at the rich diff, you might need to place all text for a given bullet on the same line. The text from line 13 is showing up on a different line from the rest of the bullet text. Same for lines 15-16